src/components/inputs/money-input/money-input.form.story.js

```diff
@@ -14,79 +15,79 @@ import Spacings from '../../spacings';
 import Readme from './README.md';
 import MoneyInput from './money-input';
 
-const validate = formValues => {
+const validate = (formValues, locale) => {
   const errors = { price: {} };
   // validate price
   if (MoneyInput.isEmpty(formValues.price)) {
     errors.price.missing = true;
-  } else if (MoneyInput.isHighPrecision(formValues.price)) {
+  } else if (MoneyInput.isHighPrecision(formValues.price, locale)) {
     errors.price.unsupportedHighPrecision = true;
   }
   return omitEmpty(errors);
 };
-
+const Story = injectIntl(props => (
```

Comment:
I moved the story to be a variable to injectItl, will point out the real changes